alicloud/resource_alicloud_slb.go

```diff
@@ -414,8 +415,10 @@ func resourceAliyunSlbUpdate(d *schema.ResourceData, meta interface{}) error {
 	if d.HasChange("specification") {
 		request := slb.CreateModifyLoadBalancerInstanceSpecRequest()
 		request.LoadBalancerId = d.Id()
+		if _, ok := d.GetOk("specification"); !ok {
+			return WrapError(Error(`'specification': required field is not set'.`))
```

Comment:
return WrapError(Error(`'specification': required field is not set when updating it'.`))